packages/modals/package.json

```diff
@@ -11,7 +11,7 @@
   "bugs": {
     "url": "https://github.com/goodhood-eu/goodhood/issues"
   },
-  "version": "0.4.0",
```

Comment:
would publish as 0.4.1 since it's kind of a fix

Reply:
fair enough